src/scripts/settings/userSettings.js

```diff
@@ -468,7 +468,8 @@ export class UserSettings {
             return this.set('enableRewatchingInNextUp', val, false);
         }
 
-        return this.get('enableRewatchingInNextUp', false);
+        const enabled = this.get('enableRewatchingInNextUp', false);
+        return enabled === 'true';
```

Comment:
Why not just
```suggestion
        return this.get('enableRewatchingInNextUp', false) === 'true';
```

Reply:
I debated going this route, but thought it might be harder to read... but it's probably fine, so let's go with that :smile: